src/main.rs

```diff
@@ -110,8 +114,7 @@ fn main() -> eyre::Result<()> {
             let current_exe =
                 current_exe().wrap_err("Unable to determine absolute path of `nix-your-shell`")?;
             if args.absolute || !executable_is_on_path(&current_exe)? {
-                shell_code =
-                    shell_code.replace("nix-your-shell", &format!("{current_exe} --absolute"));
+                shell_code = shell_code.replace("nix-your-shell", current_exe.as_str());
```

Comment:
Looks like adding `--absolute` doesn't add anything useful to the shell code, but it messes up with nushell integration

Reply:
I think I wanted `--absolute` to be reflected here:

```ShellSession
$ nix-your-shell fish env
# If you see this output, you probably forgot to pipe it into `source`:
# nix-your-shell fish | source
...

$ nix-your-shell --absolute fish env
# If you see this output, you probably forgot to pipe it into `source`:
# /Users/wiggles/.nix-profile/bin/nix-your-shell --absolute fish | source
...
```

The idea being it would show you the exact command you need to run to get the same output.

I'm not convinced it's super useful, though, so maybe we can just leave it out.

This is also straining the limits of "do a literal string replace for `nix-your-shell`" as a templating mechanism. It might be worth integrating an actual templating engine like [MiniJinja](https://lib.rs/crates/minijinja) or [tera](https://lib.rs/crates/tera) here -- what do you think? I'm loathe to lose the property of the shell snippets in `data/` being actual shell code you can run unmodified, but maybe it's fine (and it seems like it'll happen sooner or later).

Reply:
I see now and for me this is not very useful either so I suggest to remove adding `--absolute` to not overcomplicate the code.

If taking about integrating templating I'd prefer not to introduce it until there is real need for this as having shell ready code could simplify integrations in more cases.